WordPress/Classes/Models/Blog.m

```diff
@@ -18,6 +18,7 @@
 static NSInteger const ImageSizeLargeHeight = 480;
 static NSInteger const JetpackProfessionalYearlyPlanId = 2004;
 static NSInteger const JetpackProfessionalMonthlyPlanId = 2001;
+static NSInteger const BusinessPlanId = 1008;
```

Comment:
Would you mind adding `WPCom` or similar to this constant's name, to make it similar to the convention we're using for the Jetpack plans above :)?

Reply:
Done